packages/background/src/utils/constants/networks.ts

```diff
@@ -142,7 +152,7 @@ export const INITIAL_NETWORKS: Networks = {
         isCustomNetwork: false,
         enable: true,
         test: false,
-        order: 1,
+        order: 0,
```

Comment:
For me 0 is not an order number is just an array index. I would start the count in 1, but is not that important if you wanna leave it like this